lms/templates/logout.html

```diff
@@ -0,0 +1,23 @@
+{% extends "main_django.html" %}
```

Comment:
1. I'm not sure if new templates should be Mako or Django or otherwise in the platform.  We probably should be totally haphazard about it, but not sure.
2. Django templates still have work on the safe template front, simply in terms of confirming what it does and doesn't do.  
- In theory, Django templates should be safe by default, so `{{ target }}` below should be properly escaped, but I can't confirm that.
- Our js escaping helpers may not work with Django, and only with Mako, but you may not need them.
- I see `escapejs` used below, but I don't think it is necessary because it is being written until HTML, not into JavaScript, even though JavaScript will probably be reading it later.
- I have no idea if `{% trans "Signed Out" %}` gets properly escaped.
So, I'd say unless someone is going to look into the safety of Django templates, we can at least say that we know Mako templates are safe when used as prescribed.